### DISCUSSION TITLE
梅林380-6300v2挂载webdav，最高速度不超过3M

### DISCUSSION THREAD

**Discussion by sailing178** [messense/aliyundrive-webdav#457] [Q&A]
### 问题描述

梅林380-6300v2挂载webdav，最高速度不超过3M，wifi测试能跑到22M。
家里主路由同样是6300v2，由N1做旁路由挂载webdav服务，300M带宽可以跑到10几M；不在家不能检验是否是主路由性能导致的webdav网速瓶颈。

是否是主路由性能瓶颈，特此请教

### 重现步骤

打开插件，播放视频
测试文件为2G，网速在200kb/s~400kb/s之间浮动，视频无法正常播放
测试文件为20G，网速在200kb/s~2700/s左右浮动，基本上是几百KB，卡死了，播不动
WIFI网速测试，下行22M，上行5M

### 版本

最新版

### 运行平台

6300v2

### 日志

Jun  8 18:57:02 dnsmasq-dhcp[3231]: DHCPREQUEST(br0) 192.168.50.165 00:08:ca:6c:d6:39 
Jun  8 18:57:02 dnsmasq-dhcp[3231]: DHCPACK(br0) 192.168.50.165 00:08:ca:6c:d6:39 sailing

**Comment by messense** [comment]
800 MHz 双核 CPU 弱鸡， N1 是 ARM Cortex-A53 四核2GHz 没法比吧。

**Reply by quartets** [reply]
我的6300v2也一样带不动大文件。我现在是在windows电脑上开命令行运行。可以同时开几个窗口，用不同的端口支持多个账号。
